Lib9c.Miner/Miner.cs

```diff
@@ -61,21 +61,6 @@ public class Miner
                     tx => !_bannedAccounts.Contains(tx.Signer),
                 };
 
-                if (_actionTypeLoader is { } actionTypeLoader)
-                {
-                    txValidators.Add(tx =>
-                    {
-                        var nextBlockIndex = _chain.Tip.Header.Index + 1;
-                        var types = actionTypeLoader.Load(new ActionTypeLoaderContext(nextBlockIndex));
-
-                        return tx.CustomActions?.All(ca =>
```

Comment:
If it causes an issue when the `ITransaction.CustomActions` is `null`, how about checking `ITransaction.SystemAction` is not `null`?

LIke:
```csharp
return !(tx.SystemAction is null) || tx.CustomActions?.All(ca =>
```

Reply:
Amended!